README.md

```diff
@@ -31,37 +25,28 @@ If you'd like to set up your own copy of the SIMPLE database, here's what we rec
 
 1. Clone or download a copy of this repo locally onto your computer.
  
-2. Set up an environment for the python code. 
+2. Set up an environment for the Python code and install dependencies. 
```

Comment:
could we also give some instructions for just using pip?

Reply:
I think it's more important that we have pip instructions than conda ones; some environments (like Codespaces) do not come with conda.

Reply:
If someone wants to provide me with pip instructions, I'm happy to include them.  Maybe opening a dedicated  issue is the way to go here.

Reply:
Created https://github.com/SIMPLE-AstroDB/SIMPLE-db/issues/493, I think we should handle that outside this PR. Looking at how the tests run is a good starting location, but there are ways we can simplify that for the user.